### ISSUE TITLE
Node.js 4.1?

### ISSUE THREAD

**Issue by StefanScherer** [SAP/node-rfc#7]
I found your node module which seems very useful for us. Thanks for creating a Node.js binding!

I just want to ask if there are plans in the near future to support Node.js 4.1?

**Comment by bsrdjan** [comment]
With the change of `FatalException(try_catch)` to `FatalException(isolate, try_catch)` the code compiles under 4.1.1 without warnings. The basic unit test "Connection" crashes with:

```shell
FATAL ERROR: v8::HandleScope::CreateHandle() Cannot create a handle without a HandleScope
```

As soon I have free capacity I do plan to investigate and fix.

**Comment by StefanScherer** [comment]
That would be great. I found jdorner's work supporting io.js with this commit  https://github.com/jdorner/node-sapnwrfc/commit/a5eee4132117e596a8e01697a274778e833625de

Seems that v8::HandleScope is now Nan::HandleScope and some other changes.

**Comment by stefanschererseal** [comment]
Hello @bsrdjan 

in the meantime I tried to start porting your module to Node 4. My work in progress is here https://github.com/plossys/node-rfc/tree/node-4

But I'm not familiar with the Nan module. Perhaps we can collaborate on this.

**Comment by bsrdjan** [comment]
Theoretically yes, practically no capacity atm, in this year at least.

**Comment by jfernandezgersol** [comment]
Hello @bsrdjan,

like @StefanScherer, I am very interested in the project. It would be great that it has compatibility with Node.js 4+. Could you provide me some information about the building process? We have some capacity in my company and I would like to support you in order to make it compatible.

Thank you in advance.

**Comment by bsrdjan** [comment]
Building from source, for the older release, is documented here: http://sap.github.io/node-rfc/install.html#building-from-source

I see the nodejs stable version 5 released, is C++ API compatible with 4 ?

**Comment by StefanScherer** [comment]
@jfernandezgersol Currently I'm using the jdorner/node-sapnwrfc module which has Node.js 4 + 5 support.

**Comment by bsrdjan** [comment]
Versions 4.2.3 and 5.3.0 supported by master branch, Windows 64 bit builds published in branched 4.2.3 and 5.3.0.
